Source/WebKit/NetworkProcess/NetworkProcess.messages.in

```diff
@@ -55,6 +55,7 @@ messages -> NetworkProcess LegacyReceiver {
 #if PLATFORM(COCOA)
 #if HAVE(MODERN_DOWNLOADPROGRESS)
     PublishDownloadProgress(WebKit::DownloadID downloadID, URL url, std::span<const uint8_t> bookmarkData, enum:bool WebKit::UseDownloadPlaceholder useDownloadPlaceholder)
+    ClientReceivedPlaceholderURL(WebKit::DownloadID downloadID)
```

Comment:
This should be the reply and sendWithAsyncReply instead of making this a separate message type.